src/App.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
-import Calculator from './Components/Calculator';
+import { Routes, Route, Link } from 'react-router-dom';
+import Home from './Components/Home/Home';
 import './App.css';
+import Qoute from './Components/Qoute/Qoute';
+import CalculatorPage from './Components/CalculatorPage/CalculatorPage';
 
-class App extends React.Component {
-  render() {
-    return <Calculator />;
-  }
+function App() {
+  return (
+    <>
+      <header>
+        <h1>Math Magicians</h1>
+
+        <ul>
+          <li><Link to="/">Home</Link></li>
+          <li><Link to="/calculator">Calculator</Link></li>
+          <li><Link to="/quote">Quote</Link></li>
+        </ul>
```

Comment:
- Optional
Consider putting your navbar in its own component. Since react is all about splitting code into smaller reusable components, It would make much more sense if navbar had its own component.

Reply:
Noted, Thank you @muneebulrehman